src/main/java/mods/railcraft/common/blocks/aesthetics/brick/BrickTheme.java

```diff
@@ -52,8 +52,7 @@ public void initRecipes(BlockBrick block) {
     BLEACHEDBONE(RailcraftBlocks.BRICK_BLEACHED_BONE, MapColor.ADOBE) {
         @Override
         public void initRecipes(BlockBrick block) {
-            CraftingPlugin.addShapelessRecipe(RailcraftItems.BLEACHED_CLAY.getStack(), new ItemStack(Items.CLAY_BALL), new ItemStack(Items.DYE, 1, 15), new ItemStack(Items.DYE, 1, 15), new ItemStack(Items.DYE, 1, 15));
-            CraftingPlugin.addFurnaceRecipe(RailcraftItems.BLEACHED_CLAY.getStack(), new ItemStack(block, 1, 2), 0.3F);
+            CraftingPlugin.addFurnaceRecipe(new ItemStack(Blocks.BONE_BLOCK), new ItemStack(block, 1, 2), 0.3F);
```

Comment:
To an extent. Clay is much harder to come by than Bone (as it is not renewable, and it requires aqua affinity and/or respiration for efficient mining of it). This recipe uses fully renewable resources.

If expense is a concern, doubling the amount gained from Smelting might be an option. There is no way to go back to a regular bone block, so there is no risk of duplication.